docs/sources/flow/reference/components/prometheus.exporter.cadvisor.md

```diff
@@ -94,7 +94,7 @@ from `prometheus.exporter.cadvisor`:
 
 ```river
 prometheus.exporter.cadvisor "example" {
-  docker     = "unix:///var/run/docker.sock"
+  docker_host     = "unix:///var/run/docker.sock"
```

Comment:
I realize this spacing was present in the original as well, but would be good to fix it all at once :smiley: 
```suggestion
  docker_host = "unix:///var/run/docker.sock"
```

Reply:
That's the reasons why originally I haven't changed it.
Done now